### ISSUE TITLE
Document `locationOptions` 

### ISSUE THREAD

**Issue by juliogarciag** [QubitProducts/cherrytree#64]
The documentation for where to use `pushState` and other location options should be updated to explicitly say that the these options should be inside a `locationOptions` key inside the options. That will save some development time to someone else :)

**Comment by KidkArolis** [comment]
Ah! You found a bug in the code - I usually pass in a custom location object so I never noticed. I'll a add a test with a fix and release 0.7.0. For now, you'll have to use `locationOptions`.

The issue is that here I'm expecting the options to be on the main router options object (like described in the docs) and create `this.locationOptions`
https://github.com/QubitProducts/cherrytree/blob/3ebe31eb73ed6ff25c79abcb8910003e4ce1cd1a/router.js#L39

But then here I read `this.options.locationOptions` instead of `this.locationOptions` (which is the bug).
https://github.com/QubitProducts/cherrytree/blob/3ebe31eb73ed6ff25c79abcb8910003e4ce1cd1a/router.js#L80

In other words, `locationOptions` is meant to be an internal variable, I'm accidentally reading it from the wrong place.

Sorry about that! And thanks for reporting.

**Comment by juliogarciag** [comment]
Awesome. Also, thanks for the great work. I'm trying to use less full-stack frameworks like ember or angular and libraries like cherrytree helps a lot :)

**Comment by KidkArolis** [comment]
Glad to hear!

**Comment by nelsonmenezes** [comment]
Might be an idea to have the updated docs for now; spent a while trying to figure this out!

**Comment by KidkArolis** [comment]
This has been fixed and released as 0.7.0. Sorry for the loooong delay. Let me know if you have any further issues with this.
